lib/vendor/de77/BMP.php

```diff
@@ -46,7 +46,7 @@ public static function imagebmp(&$img, $filename = false)
 		$hei     = imagesy($img);
 		$wid_pad = str_pad('', $wid % 4, "\0");
 
-		$size = 54 + ($wid + $wid_pad) * $hei * 3; //fixed
+		$size = 54 + ($wid + strlen($wid_pad)) * $hei * 3; //fixed

```

Comment:
Yeah, your formula in https://github.com/smottt/WideImage/pull/18#issuecomment-757241933 was correct, the `* 3` needs to be moved:

```php
		$size = 54 + ($wid * 3 + strlen($wid_pad)) * $hei; //fixed

```